env.example

```diff
@@ -143,14 +143,18 @@ ETHERPAD_SKIN_VARIANTS="super-light-toolbar super-light-editor light-background
 # LDAP authentication (for more information see the Cyrus SASL saslauthd.conf man page)
 #
 
-# LDAP url for connection
-#LDAP_URL=ldaps://ldap.domain.com/
+# LDAP url:port for connection
```

Comment:
as far as I know specifying the port as no effect. Did you check ?

Reply:
cyrus' docs are somewhat indecisive, see below. I changed L:146 back to its original state

```
ldap_servers: <ldap://localhost/>
	Specify URI(s) refering to LDAP server(s), e.g. ldaps://10.1.1.2:999/.
	You can specify multiple servers separated by a space.
```